spec/spec_helper.rb

```diff
@@ -6,4 +6,12 @@
   config.expect_with :rspec do |expectations|
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
+
+  config.backtrace_exclusion_patterns = [
+    /\/lib\d*\/ruby\//,
+    /bin\//,
+    /gems/,
+    /spec\/spec_helper\.rb/,
+    /lib\/rspec\/(core|expectations|matchers|mocks)/
+  ]
```

Comment:
Sorry I meant to mention that - it removes the rvm stuff from test stack
traces
On May 14, 2016 3:30 PM, "Josh Washke" <notifications@github.com> wrote:

> In spec/spec_helper.rb
> <https://github.com/patrickwhardy/the_pivot/pull/54#discussion_r63282318>:
>
> > @@ -6,4 +6,12 @@
> >    config.expect_with :rspec do |expectations|
> >      expectations.include_chain_clauses_in_custom_matcher_descriptions = true
> >    end
> > +
> > +  config.backtrace_exclusion_patterns = [
> > +    /\/lib\d*\/ruby\//,
> > +    /bin\//,
> > +    /gems/,
> > +    /spec\/spec_helper\.rb/,
> > +    /lib\/rspec\/(core|expectations|matchers|mocks)/
> > +  ]
>
> Whats this code do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/patrickwhardy/the_pivot/pull/54/files/7283292b5fe76e16c30024bce640a2a39be3e4c2#r63282318>
>